extensions/sonos/src/decrease-volume.tsx

```diff
@@ -30,5 +30,11 @@ export default async function Command() {
     });
   } else {
     await coordinator.SetRelativeGroupVolume(-Number(preferences.volumeStep));
+    await coordinator.GetState().then((state) =>
```

Comment:
If I remember correctly, the new volume is actually returned from `SetRelativeGroupVolume`. Couls you try that?

Reply:
you're right. it does return the new volume.